VERSION

```diff
@@ -1 +1 @@
-1.0.0
```

Comment:
Not sure if `2.0.0` is a good idea. We might want to release a `1.90.0` or similar if some tools can't cope with `2.0.0-alpha1`. Maybe we should set this to a high 1.x number for now

Reply:
So this means that you may want to test the new script? ;-)

Reply:
@dkroenke Could we just omit setting alpha and rc releases for colcon?

Reply:
@mossmaurice 
I prefer to be consistent with the version numbers in all places because it could lead to confusion when the cmake files and package.xml files are showing different version numbers (Which one is the truth?).
With the script it is not a big deal anymore to change the version numbers. And you never know beforehand if a bugfix release may get into ROS.